pinot-core/src/main/java/org/apache/pinot/core/plan/TransformPlanNode.java

```diff
@@ -124,6 +143,10 @@ public TransformOperator run() {
     return new TransformOperator(_projectionPlanNode.run(), new ArrayList<>(_expressionTrees));
   }
 
+  public List<FieldSpec.DataType> getProjectedColumnTypes() {
```

Comment:
This should come from metadata, and not from PlanNode, right?

Reply:
I see, but as mentioned in the response above https://github.com/apache/incubator-pinot/pull/4535#discussion_r315381929, this projected column type info is being used later to build DataTable in InstanceResponseBlock